models/stg_google_play__stats_ratings_app_version.sql

```diff
@@ -0,0 +1,35 @@
+
+with base as (
+
+    select * 
+    from {{ ref('stg_google_play__stats_ratings_app_version_tmp') }}
+
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_google_play__stats_ratings_app_version_tmp')),
+                staging_columns=get_stats_ratings_app_version_columns()
+            )
+        }}
+        
+    from base
+),
+
+final as (
+    
+    select 
+        date as date_day,
+        app_version_code,
+        package_name,
+        -- an average of an average will only be taken for NULL app versions :-) as they are being grouped together
+        avg( cast( nullif(daily_average_rating, 'NA') as {{ dbt_utils.type_float() }} )) as average_rating,
```

Comment:
Would a `case when` also suffice here? this `nullif` case isn't explicitly stated in our [style guide](https://github.com/fivetran/dbt_style_guide/blob/main/sql_guidelines.md#general-sql-syntax), but wouldn't hurt to add it. I don't think there's anything wrong with this, but I did have to look it up to make sure my understanding was correct for the function.

Reply:
a `case when` could work but `nullif` is definitely more concise, open to opinions tho

Reply:
i think i tend to opt for more readable code and would prefer a case when, but let me add this to the agenda for discussion later!